### PR TITLE
Added info. Closes #49, closes #65, (finishing) closes #55

### LINKED ISSUE #49: Add more employee information to onboarding
- [x] Phone contact
- [x] Emergency contact name and number
- [ ] Hire Date
- [x] Hook up Locations and Departments

### LINKED ISSUE #55: Add locations

### LINKED ISSUE #65: [Feature] Supervisor tracking
- [ ] Add selection in employee onboarding
- [ ] display in employee information
- [ ] display all `supervises` in employee information

### PR DESCRIPTION
**Pull request by jscottchapman**
Changed payRateHourly and payRateYearly and payRateMonthly to payRate.
Created format-phone helper.
Added phone contact, emergency contact info, and added locations and
departments.

### CODE REVIEW

**Comment by jscottchapman** [comment]
@James1x0   Do I need to do anything else to this before it can be merged?

File: app/styles/app.less [hunk 1]
```diff
@@ -5,3 +5,4 @@
 @import "signup.less";
 @import "granite/page/email-sent.less";
 @import "granite/page/about.less";
+@import "granite/page/employee.less";
```
Comment (James1x0) [inline app/styles/app.less]: This line isn't needed as we use less globbing to import whole trees.

File: app/routes/account/index.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default Route.extend({
       }
     };
     if(!Ember.isEmpty(params.tag)) {
-      activityQuery.tag = params.tag;
+      activityQuery.tag = { $in: params.tag.split(',') };
     }
```
Comment (James1x0) [inline app/routes/account/index.js]: Could you add spacing around L20 after L19 & L22

File: app/styles/granite/global/utility.less [hunk 3]
```diff
@@ -45,3 +45,19 @@
...
+  transition: all ease-in-out 0.35s;
+
+  &:hover {
+    background-color: lighten(@granite-green, 15%);
+    box-shadow: 0 3px 7px fade(@granite-grey, 65%);
+    }
+}
+
+.vertically-aligned {
+
```
Comment (James1x0) [inline app/styles/granite/global/utility.less]: Extra line here.

File: app/styles/granite/global/utility.less [hunk 4]
```diff
@@ -45,3 +45,19 @@
...
+.shadow-hover {
+  box-shadow: 0 3px 6px fade(@granite-grey, 70%);
+  z-index: 1;
+  transition: all ease-in-out 0.35s;
+
+  &:hover {
+    background-color: lighten(@granite-green, 15%);
+    box-shadow: 0 3px 7px fade(@granite-grey, 65%);
+    }
```
Comment (James1x0) [inline app/styles/granite/global/utility.less]: Funky curly brace tab here.

File: app/templates/account/employee/index/equipment.hbs [hunk 5]
```diff
@@ -12,5 +12,8 @@
     {{/if}}
   </div>
 {{else}}
-  <div class="header">No Items Assigned, Yet!</div>
+  <div class="ui grey segment">
+    <h2 class="header">No Items Assigned, Yet!</h2>
+    <h2 class="header">Assign some now<i class="big plus icon"></i></h2>
```
Comment (James1x0) [inline app/templates/account/employee/index/equipment.hbs]: This plus icon doesn't really do anything as it seems. Let's note to fix this in the future as we may indicate here, to a user, that you can click it to add something.

File: app/templates/account/employee/index/index.hbs [hunk 6]
```diff
@@ -9,26 +9,26 @@
         </a> <br />
         <a href="tel:{{model.phone}}">
           <i class="phone icon"></i>
-          {{model.phone}}
+          {{format-phone model.phone}}
         </a>
       </div>
     </div>
-    <div class="ui raised segment">
+    <div class="ui raised teal segment">
```
Comment (James1x0) [inline app/templates/account/employee/index/index.hbs]: Straight swag 👍

**Comment by James1x0** [comment]
@jscottchapman rebase from canary plz. 🙏

**Comment by jscottchapman** [comment]
@James1x0 Should be good to go now.

**Comment by James1x0** [comment]
@jscottchapman Is this only closing #49 btw or are there any other issues implicated?

**Comment by jscottchapman** [comment]
@James1x0 Probably #65 and #55 as well.

**Comment by James1x0** [comment]
@jscottchapman Agreed besides item 3 in #65

**Pull request merged**